src/App.js

```diff
@@ -1,8 +1,13 @@
 import "./App.scss";
-import LogIn from "./pages/Login/LogIn";
+import SideBarMenu from "./components/GeneralMenu/SideBarMenu";
+import NavBarMenu from "./components/GeneralMenu/NavBarMenu";
 
 function App() {
-  return <div className="App"></div>;
+  return <div className="App">
+     <NavBarMenu/>
```

Comment:
Leave the App.js empty components should be used here. This space is only for rendering Pages